vault/dendron.topic.special-notes.md

```diff
@@ -2,113 +2,71 @@
 id: 5c213aa6-e4ba-49e8-85c5-1bdcb33ce202
 title: Special Notes
 desc: ''
-updated: 1660887018450
+updated: 1661500585942
 created: 1595004457029
+config:
+    global:
+        enableChildLinks: false
 ---
 
-Dendron has built-in support for a variety of special note formats. These notes can be created using regular lookup - these commands provide convenient shortcuts for frequently used notes.
+## Summary
 
-## Journal Note
+Dendron comes with built-in special notes that could be used out of the box for variety of use cases.
 
-A journal note is a self contained note that is meant to track something over time. Examples of journals include recording **workout sessions**, making **meeting notes**, and keeping a **mood journal**.
+## Features
 
-To create a journal note, trigger a lookup and then click on the calendar icon, or use the `Ctrl+Shift+J`/`Cmd+Shift+J` shortcut (depending on operating system).
+### Start writing notes instantly
 
-<a href="https://www.loom.com/share/3c3ddc1dc63547cea8bf186bec31f71b"> 
-<img style="" src="https://cdn.loom.com/sessions/thumbnails/3c3ddc1dc63547cea8bf186bec31f71b-with-play.gif"> </a>
+With special notes, you can choose a specific type of note that fits your use case and start writing notes instantly without thinking about the format or structure.
 
-By default, Dendron will create the journal note with the following hierarchy `{domain}.journal.{y.MM.dd}`. `{domain}` is the **domain** of the current active note when you execute `New Journal Note`.
+### Easily create and open frequently used notes
 
-A reference for date formatting tokens can be found [here](https://moment.github.io/luxon/#/formatting)
+Each special note comes with dedicated commands that help you navigate / create and manage them.
+They could also be creating using lookup by applying [[lookup modifiers|dendron.topic.lookup.modifiers]]
 
-## Scratch Note
+### Expand based on your use case
```

Comment:
would modify to 

Builtin Best Practices

Special notes come with builtin schemas and templates. You can use them as is or customize for your own needs.